src/05-objects.js

```diff
@@ -35,7 +43,12 @@ function createObjectWithValue(val) {}
  *  createObjectWithKey("left");
  *  //> { left: false }
  */
-function createObjectWithKey(key) {}
+function createObjectWithKey(key) {
+  let newObj = new Object();
+    newObj[key]= 'milk'
+  
```

Comment:
Lets get back to our indentation and also I don't think we need an empty line between our return statements.